File: MaskInfo/app/src/main/java/com/example/maskinfo/repository/MaskService.java
package com.example.maskinfo.repository;

import com.example.maskinfo.model.StoreInfo;

import retrofit2.Call;
import retrofit2.http.GET;

public interface MaskService {
    String BASE_URL = "https://8oi9s0nnth.apigw.ntruss.com/corona19-masks/v1/";

    @GET("storesByGeo/json/")
    Call<StoreInfo> fetchStoreInfo();
}

File: MaskInfoKotlin/app/src/main/java/com/example/maskinfokotlin/repository/MaskService.kt
package com.example.maskinfokotlin.repository

import com.example.maskinfokotlin.model.StoreInfo
import retrofit2.Call
import retrofit2.http.GET
import retrofit2.http.Query

interface MaskService {
    companion object {
        const val BASE_URL = "https://8oi9s0nnth.apigw.ntruss.com/corona19-masks/v1/"
    }

    @GET("storesByGeo/json/?m=5000")
    fun fetchStoreInfo(
        @Query("lat") lat: Double,
        @Query("lng") lng: Double
    ): Call<StoreInfo>
}